proto/axelar/multisig/v1beta1/query.proto

```diff
@@ -34,6 +34,16 @@ message NextKeyIDResponse {
   ];
 }
 
+message KeyEpochRequest {
+  string chain = 1;
+  uint64 epoch = 2;
+  bool current = 3;
+}
+
+message KeyEpochResponse {
+  KeyEpoch key_epoch = 1 [ (gogoproto.customname) = "KeyEpoch" ];
+}
```

Comment:
Can you please also add a REST API endpoint for this in `service.proto` and a CLI query?